lib/system/dragonbox.nim

```diff
@@ -1039,19 +1043,6 @@ proc toDecimal64*(ieeeSignificand: uint64; ieeeExponent: uint64): FloatingDecima
 ## ==================================================================================================
 
 proc utoa2Digits*(buf: var openArray[char]; pos: int; digits: uint32) {.inline.} =
-  const
```

Comment:
This is also duplicated in schubfach.nim

Reply:
I see